frontend/src/axios.ts

```diff
@@ -0,0 +1,38 @@
+import { default as ax } from 'axios';
+import camelcaseKeys from 'camelcase-keys';
+import decamelizeKeys from 'decamelize-keys';
+
+const axios = ax.create({
+  baseURL: 'http://0.0.0.0:8000/api/v1',
```

Comment:
It will be better to use `backendUrl` from a config.tsx file

Reply:
sure, i was wondering whether we actually deploy this during presentation, if not i would stick to localhost or else we need to introduce feature flag/ if with environment based mapping